### ISSUE TITLE
Mulit Level Applications Menu

### ISSUE THREAD

**Issue by junland** [ritenv/retro-desktop#1]
Hello! Just wanted to point out that there's no multi level applications menu when you press the main menu button. All around it's a awesome project! Hope there's more down the pipeline! :)

**Comment by ozznixon** [comment]
@ritenv agreed an example or script to do main menus on the window, and another of how to do a toolbar on a window (assuming under the menu). I am looking to make a "Fake OS" web game... so far this is an awesome (and tight) example for a retro framework. (previously I was using DHTMLX and before that DOMAPI - this is much smaller.

**Comment by ritenv** [comment]
This is an old project built 8 years ago just for fun, and is not being maintained by me in any way. Feel free to take it forward as needed! :)

**Comment by ozznixon** [comment]
Could you help me make a start UXToolbar.js? I can take it from there and
do the css, etc. Just not sure how to start it. (I am a true Vanilla js
developer and never used jquery for anything)...

Thanks,
Ozz
